### DISCUSSION TITLE
能否尝试接入百度翻译的领域翻译api

### DISCUSSION THREAD

**Discussion by miguo4god** [YiiGuxing/TranslationPlugin#4016] [Ideas]
百度翻译开放了领域翻译的api，是否考虑尝试接入呢

**Comment by YiiGuxing** [comment]
它那个需要高级会员才能用，我没有高级会员，无法开发调试。

**Reply by BBQPYJS** [reply]
百度翻译，语音无法播放，提示TTS连接超时。

**Reply by YiiGuxing** [reply]
> 百度翻译，语音无法播放，提示TTS连接超时。

@BBQPYJS TTS 目前都是谷歌的，国内不能直接使用。
